### ISSUE TITLE
Compressed Skeleton's doesn't have bows...

### ISSUE THREAD

**Issue by SWEPatA** [blay09/ExCompressum#64]
Compressed Skeleton's doesn't have bows, bug or feature?

Also Compressed Whiter's maybe?

**Comment by blay09** [comment]
Clearly a feature, since only the Skeleton was compressed, not the bow.

**Comment by SWEPatA** [comment]
Let me rephrase, Compressed Skeleton's has boss but the Decompressed Skeleton's don't have.

**Comment by blay09** [comment]
I know what you mean. I might give them a bow in a future version, I just couldn't be bothered. :P
